src/SplineExtrapolations/SplineExtrapolations.jl

```diff
@@ -30,6 +32,26 @@ function extrapolate_at_point(::Flat, S::Spline, x)
     S(x′)
 end
 
+"""
```

Comment:
Thanks for adding a docstring. We should also add `Linear` to the docs/src/extrapolation.md file (next to the other extrapolation methods), so that the docstring is displayed in the deployed docs.

Reply:
Added